lib/matplotlib/tests/test_backend_macosx.py

```diff
@@ -18,3 +22,26 @@ def test_cached_renderer():
     fig = plt.figure(2)
     fig.draw_without_rendering()
     assert fig._cachedRenderer is not None
+
+
+@pytest.mark.backend('macosx')
+def test_savefig_rcparam(monkeypatch):
+
+    def new_choose_save_file(title, directory, filename):
```

Comment:
This test should maybe assert that the directory is in the expected temporary directory as well? (And maybe you want to use the `tmp_path` fixture instead of putting everything in a `tempfile` context)

Reply:
More good suggestions :) Done.